api/all.yaml

```diff
@@ -2097,6 +2097,21 @@ components:
           type: integer
           format: int64
       type: object
+    UserResource:
+      description: SCIM provides a resource type for "User" resources.
+      type: object
+      properties:
+        userName:
+          description: Unique identifier for the User.
+          type: string
+        name:
```

Comment:
Double check the fields that we process. I made some changes like adding `displayName` (which maps to the entity `name` field).